src/services/expenses.services.js

```diff
@@ -0,0 +1,47 @@
+const { getExpensesFilterQuery } = require('../utils/expenses');
+
+const { Expense } = require('../models/Expense.model');
+
+const getExpenses = async (categories, userId, from, to) => {
+  const where = getExpensesFilterQuery(categories, userId, from, to);
+
+  const expenses = await Expense.findAll({ where });
+
+  return expenses;
+};
+
+const getExpenseById = async (id) => {
+  const expense = await Expense.findByPk(id);
+
+  return expense;
+};
+
+const createExpense = async (expense) => {
+  const newExpense = await Expense.create(expense);
+
+  return newExpense;
+};
+
+const removeExpense = async (id) => {
+  await Expense.destroy({
+    where: {
+      id,
+    },
+  });
+};
+
+const updateExpense = async (id, valuesToUpdate) => {
+  await Expense.update(valuesToUpdate, { where: { id } });
```

Comment:
Consider checking the result of the `update` operation to ensure that the update was successful. This can help identify cases where the expense ID does not exist.